region/maridia/outer.json

```diff
@@ -3088,9 +3278,73 @@
                     "Run and jump from on top of the crab to the higher platform to the left. A failure often requires resetting the room."
                   ],
                   "devNote": "Although this goes through 7, it requires starting at 2 to get the crab to move into position."
+                },
+                {
+                  "name": "Cross Room Jump - HiJump Space Jump",
```

Comment:
SpringballJump is another way to get a jump boost near the transition.  All jumpways can reach 8 except for croc room which needs a crouch jump.
Then with hijump you can reach 11, but hijump+springball can do that from 8 already.